sonic_data_client/events_client.go

```diff
@@ -100,7 +109,11 @@ func NewEventClient(paths []*gnmipb.Path, prefix *gnmipb.Path, logLevel int) (Cl
                     log.V(7).Infof("evtc.heartbeat_interval is set to %d", val)
                     Set_heartbeat(val)
                 }
-                break
+            } else if (k == PARAM_QSIZE) {
+                if val, err := strconv.Atoi(v); err == nil {
+                    evtc.pq_max = val
```

Comment:
What is the limit for PQ Max size? If someone passes in INT_MAX, is that acceptable?

Reply:
Good catch
Let me add a max

Reply:
done